components/org.wso2.carbon.identity.application.authenticator.push/src/main/java/org/wso2/carbon/identity/application/authenticator/push/notification/handler/RequestSender.java

```diff
@@ -39,6 +39,7 @@
      * @param key      Session Data Key
      * @throws PushAuthenticatorException if an error occurs while preparing the push notification
      */
-    void sendRequest(HttpServletRequest request, HttpServletResponse response, String deviceId, String key)
+    void sendRequest(HttpServletRequest request, HttpServletResponse response, String deviceId, String key,
```

Comment:
Lets overload this method as `additionalInfo` seems to be optional

Reply:
In the impl class simply call the overloaded method from this method passing `additionalInfo` as null (without duplicating the code)

Reply:
Fixed with https://github.com/wso2-extensions/identity-outbound-auth-push/pull/118/commits/061f61e9ccdcddd46a1a152bc5f126ed13cbd1aa